GPflow/tf_wraps.py

```diff
@@ -20,13 +20,16 @@
 import os
 import tensorflow as tf
 from tensorflow.python.framework import ops
+import warnings
 from ._settings import settings
+import numpy as np
 
 
 def eye(N):
     """
     An identitiy matrix
     """
+    warnings.warn('tf_wraps.eye is deprecated: use tf.eye instead', np.VisibleDeprecationWarning)
```

Comment:
So we don't just want to take it out and let people change their code? Typically before version 1.0 code bases reserve the right to make API-breaking changes. I know it's only a tiny piece of code, but it might be good to not let GPflow suffer from deprecation-bloat.